bolt/include/bolt/Rewrite/DWARFRewriter.h

```diff
@@ -89,6 +89,15 @@ class DWARFRewriter {
   /// Store Rangelists writer for each DWO CU.
   RangeListsDWOWriers RangeListsWritersByCU;
 
+  /// Contains ranges writer and die.
+  struct RangesWriterAndDIE {
+    std::unique_ptr<DebugRangesSectionWriter> Writer;
+    DIE *Die;
```

Comment:
If we expect these two to be tied together, maybe make `Die` an optional field in `DebugRangesSectionWriter`?